monitoring/README.md

```diff
@@ -58,12 +60,14 @@ thanos-storegateway-0                                  1/1     Running   0
 ```
 
 
-b) Deploy the monitoring tools in a WEC cluster (e.g., `cluster1`):
+b) Deploy the monitoring tools in a plain K8s WEC cluster (e.g., `cluster1`):
 
 ```bash
 ./install-ks-monitoring.sh --cluster-context cluster1 --set wec
 ```
 
+For OpenShift clusters add the flags `--env ocp` & `--hub-context <hub-context-name>` in the commad above. 
```

Comment:
In KubeStellar terms, is this a context for the KubeFlex hosting cluster or the ITS?

Reply:
The flag `--cluster-context`  refers to either the context of the hosting KubeFlex cluster OR of a WEC cluster - depending when the monitoring tooling is being deployed.

Reply:
Sorry, I was not clear. My question is not about `--cluster-context`. My question is about `--hub-context`. Does this mean the KubeFlex hosting cluster or does it mean a KubeStellar ITS?

Reply:
`--hub-context` means the KubeFlex hosting cluster.

Reply:
FYI, for better clarity, I replaced the flag name `--hub-context` with `kubeflex-hosting-cluster-context`.